trnsysGUI/SaltTank.py

```diff
@@ -0,0 +1,138 @@
+# pylint: skip-file
+# type: ignore
+
+import os
+import shutil
+import typing as _tp
+
+from PyQt5.QtWidgets import QTreeView
+
+import trnsysGUI.createSinglePipePortItem as _cspi
+import trnsysGUI.images as _img
+import trnsysGUI.massFlowSolver.networkModel as _mfn
+from trnsysGUI.BlockItem import BlockItem
+from trnsysGUI.MyQFileSystemModel import MyQFileSystemModel
+from trnsysGUI.MyQTreeView import MyQTreeView
+from trnsysGUI.massFlowSolver import InternalPiping, MassFlowNetworkContributorMixin
+
+
+class SaltTank(BlockItem, MassFlowNetworkContributorMixin):
```

Comment:
Good idea with the base class. Let's also make it abstract, i.e. inherit from `_abc.ABC` (`import abc as _abc`) ["abc" = "Abstract base class"].

Reply:
What changes would this imply for this specific case?